nx-cloud/templates/cloud.yml

```
@@ -982,21 +982,13 @@ spec:
     spec:
       containers:
         - name: nx-cloud-messagequeue
-          image: nsqio/nsq:v1.2.0
+          image: nxprivatecloud/nx-cloud-messagequeue:latest
```

should we use the version they specify? (so we don't break anything for them if they chose not to upgrade)

```suggestion
          image: nxprivatecloud/nx-cloud-messagequeue:{{ .Values.image.tag }}
```